include/fmt/core.h

```diff
@@ -395,9 +395,9 @@ FMT_CONSTEXPR auto to_unsigned(Int value) ->
   return static_cast<typename std::make_unsigned<Int>::type>(value);
 }
 
-FMT_MSC_WARNING(suppress : 4566) constexpr unsigned char section[] = "\u00A7";
+FMT_CONSTEXPR auto is_utf8() -> bool {
```

Comment:
You need `inline` here because `FMT_CONSTEXPR` expands to nothing on compilers that don't support relaxed constexpr.